cms/static/sass/elements/_modal.scss

```diff
@@ -181,7 +181,91 @@
         text-shadow: 0px 1px 0px #F9F8F8;
         background-clip: padding-box;
         font-size: 0.8125em;
+        width: 100%;
```

Comment:
Remove this new code from the modals partial when you move the form into the page frame.